### PR TITLE
[翻译更新] utilities.po

### PR DESCRIPTION
**Pull request by isdanni**
Update according to [#959 ](https://github.com/taichi-dev/taichi/pull/959) in Taichi dev repo;

Issue: #38

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 1]
```diff
@@ -58,40 +56,46 @@ msgid ""
...
 msgstr ""
+"运行 ``ti baseline`` 以将基准结果保存到 ``benchmarks/baseline`` 以供将来"
+"比较，这可以在与性能有关的PR上执行，然后再合并到master中。"
 
 #: ../../utilities.rst:35
 msgid ""
 "For example, this is part of the output by ``ti regression`` after "
 "enabling constant folding optimization pass:"
-msgstr ""
+msgstr "例如，这是启用恒定折叠优化传递后 ``ti regression`` 输出的一部分："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
msgstr "例如，这是启用常数折叠优化传递后 ``ti regression`` 输出的一部分："
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 2]
```diff
@@ -107,64 +111,82 @@ msgid ""
...
+"取决于硬件性能，因此如果基准来自另一台计算机则很难进行比较。 我们将在某个"
+"时间点购买固定性能的机器作为时间基准服务器。 讨论在: https://github.com/"
+"taichi-dev/taichi/issue/948"
 
 #: ../../utilities.rst:58
 msgid ""
 "The suggested workflow for **the PR author** to run the regression tests "
 "is:"
-msgstr ""
+msgstr "建议 **the PR author** 运行回归测试的工作流程是："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po:144]: Not translated? Btw, this part is recently updated, so keep this untranslated before `make merge` is OK.

**Review by archibate — APPROVED** [review]
Sorry for the delayed reply. Thank for updating this! Btw, I decide to organize a translation team for the repo with long future update plan, would you like to participate?

File: locales/zh_CN/LC_MESSAGES/utilities.po (hunk 2 above)
Comment (isdanni) [reply locales/zh_CN/LC_MESSAGES/utilities.po:144]: Don't know how I missed that lol ...
Well noted :-)

**Comment by isdanni** [comment]
> Sorry for the delayed reply. Thank for updating this! Btw, I decide to organize a translation team for the repo with long future update plan, would you like to participate?

Sure I'd love to! Thanks for the review :-)

**Pull request merged**